packages/ccss/ccss.1.6/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
+authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
+homepage: "http://ccss.forge.ocamlcore.org/"
+bug-reports: "https://github.com/darioteixeira/ccss/issues"
+dev-repo: "https://github.com/darioteixeira/ccss.git"
```

Comment:
If you want this to be generated by `oasis2opam`, add a `SourceRepository` section to your `_oasis` file.